pkg/drivers/k8scel/driver.go

```diff
@@ -15,7 +15,7 @@ import (
 	"github.com/open-policy-agent/frameworks/constraint/pkg/types"
 	pSchema "github.com/open-policy-agent/gatekeeper/v3/pkg/drivers/k8scel/schema"
 	"github.com/open-policy-agent/gatekeeper/v3/pkg/drivers/k8scel/transform"
-	"github.com/open-policy-agent/opa/storage"
```

Comment:
does using v1/storage satisfy the interface? IIRC, constraint framework change is pointing to v0 to start.

Reply:
reverted this to be consistent with frameworks